Sources/GRPC/Shims.swift

```diff
@@ -288,3 +289,44 @@ public struct GRPCTimeoutError: Error, Equatable, CustomStringConvertible {
   /// the gRPC wire-format.
   public static let tooManyDigits = GRPCTimeoutError(.tooManyDigits)
 }
+
+
+extension UnaryCallHandler {
+  @available(*, deprecated, message: "Please regenerate your code or use 'CallHandler.makeUnary'")
```

Comment:
Maybe it's better to just mark this `unavailable` instead of `deprecated`, given that it explodes anyway?

Reply:
Didn't realise that was err.. available. Thanks for the tip.